src/services/device-management/lorawan-device-database-enrich-job.ts

```diff
@@ -69,6 +80,19 @@ export class LorawanDeviceDatabaseEnrichJob {
     );
   }
 
+  @Cron(CronExpression.EVERY_DAY_AT_4AM)
+  async checkUnusualPackagesForGateways() {
+    const gateways = await this.gatewayService.getAllWithUnusualPackagesAlarms();
+    try {
+      for (let index = 0; index < gateways.resultList.length; index++) {
```

Comment:
Minor: For den ene alarm funktion ahr du en wrapper funktion til foreach i gateway servicen, for den andne gør du det her